ros/processor/inventory_events_consumer.py

```diff
@@ -167,3 +170,10 @@ def process_system_details(self, msg):
                 ).inc()
                 LOG.error("%s - Unable to add system %s to DB belonging to account: %s and org_id: %s - %s",
                           self.prefix, host['fqdn'], account.account, account.org_id, err)
+
+
+if __name__ == "__main__":
+    start_http_server(int(METRICS_PORT))
+    commence_cw_log_streaming('ros-processor')
```

Comment:
likewise, can we have different logging stream?

Reply:
Done